pkg/app/piped/executor/waitapproval/waitapproval.go

```diff
@@ -114,3 +115,27 @@ func (e *Executor) checkApproval(ctx context.Context) (string, bool) {
 	}
 	return approveCmd.Commander, true
 }
+
+func (e *Executor) reportRequiringApproval(ctx context.Context) {
+	ds, err := e.TargetDSP.GetReadOnly(ctx, e.LogPersister)
+	if err != nil {
+		e.LogPersister.Errorf("Failed to prepare running deploy source data (%v)", err)
+	}
+
+	var approvers []string
+
+	for _, v := range ds.GenericDeploymentConfig.DeploymentNotification.Mentions {
+		if v.Event == "WATING_APPROVAL" {
```

Comment:
This should be `DEPLOYMENT_WAIT_APPROVAL`.

Reply:
132a66f